APCAppCore/APCAppCore/Library/Categories/NSError+Bridge.m

```diff
@@ -87,7 +87,13 @@ - (NSString*)bridgeErrorMessage
 		}
 		else if([errors valueForKey: kSageErrorPasswordKey])
 		{
-			message = kAPCBadPasswordErrorMessage;
+            NSArray *passwordErrors = [errors valueForKey: kSageErrorPasswordKey];
+            NSString *errorsString = [passwordErrors componentsJoinedByString:@", "];
+            if (errorsString.length > 0) {
+                message = [NSString stringWithFormat:@"%@ Details: %@", kAPCBadPasswordErrorMessage, errorsString];
```

Comment:
Please localize, e.g.:

NSString *format = NSLocalizedStringWithDefaultValue(@"APC_ERROR_BAD_PASSWORD_FORMAT", @"APCAppCore", APCBundle(), @"The password you have entered is not a valid password.  Please try again. Details: %@", @"Format for error message when participant attempts to sign up with a non-valid password, to be filled in with specifics of why it's not valid from the server error message");
message = [NSString stringWithFormat:format, errorsString];